massgen/MULTI_SOURCE_AGENT_INTEGRATION_DESIGN.md

```diff
@@ -1294,9 +1626,9 @@ def load_framework_agent(config: Dict[str, Any]) -> ChatAgent:
 massgen/
 ├── adapters/                      # All framework adapter implementations
 │   ├── __init__.py               # Export main adapter classes
-│   ├── base.py                   # Base FrameworkAgentAdapter class
-│   ├── converters.py             # MessageConverter and ToolConverter classes
-│   ├── factory.py                # FrameworkAdapterRegistry and load_framework_agent
+│   ├── base.py                   # Base FrameworkAdapter class
+│   ├── registry.py               # Adapter registry and registration functions
```

Comment:
I like the registry part.
Massgen still lack of toolkit registry for backends. 
Good point!